include/bout/fv_ops.hxx

```diff
@@ -194,7 +193,8 @@ namespace FV {
 
     Coordinates *coord = f_in.getCoordinates();
 
-    Field3D result = 0.0;
+    Field3D result{emptyFrom(f_in)};
+    result = 0.0;
```

Comment:
We actually do this a fair bit, use `emptyFrom` and then set to 0.0. Is it worth having another helper function like `zeroFrom(f)`?

Reply:
How about another modifier method? So we could write something like `Field3D result{emptyFrom(f_in).fillZeros();}?

Reply:
...on the other hand, I think `zeroFrom` would take less code to implement, and require less typing to use. Seems like a good reason to add a new function instead of more methods :+1:

Reply:
Yeah, I think another free function is better than a method. It looks like we currently only need a `zeroFrom(Field)` rather than a more generic `fillFrom(BoutReal, Field)`?

Reply:
Yep, all the places we use `emptyFrom` we only ever set to zero afterwards, not to any non-zero value.